server/app/swagger.json

```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.0",
   "info": {
     "title": "Open CO2 REST API",
-    "version": "0.3.0"
+    "version": "0.5.0"
```

Comment:
Good point 👍 Maybe that the command to regenerate the swagger file could be played after the `release` one to prevent forgetting it…